### PR TITLE
Add a debug server

### LINKED ISSUE #12: Add a way to test in localhost
Currently `x.py` only allow to build the website or to deploy it with SSH on the `caniuse.rs` server.

it would be great to be able to server it locally, eg `x.py run 8000` to serve it on http://localhost:8000.

### PR DESCRIPTION
**Pull request by a1phyr**
Closes #12 

Add a `serve` subcommand to `x.py` to easily use the site in localhost.

I did not implement a way to choose the port.

### CODE REVIEW

File: x.py [hunk 1]
```diff
@@ -46,6 +55,14 @@ def build():
     )
 
 
+def serve():
+    from http import server
+
+    build()
+    handler = functools.partial(server.SimpleHTTPRequestHandler, directory="public/")
+    server.test(handler, bind='127.0.0.1')
```
Comment (jplatte) [inline x.py:63]: I've had a look at `http.server` docs, and don't see `test` anywhere? The closest example I can find it right above [here](https://docs.python.org/3/library/http.server.html#http-server-cli), and the other ones I see also use `serve_forever`.

**Review by jplatte — COMMENTED** [review]
Thanks for the PR! 🙂

File: x.py (hunk 1 above)
Comment (a1phyr) [reply x.py:63]: I found it in the source of `http.server`: https://github.com/python/cpython/blob/3.8/Lib/http/server.py#L1238
It is the function used when doing `python -m http.server`.

If you prefer something more "official", I can change that.

Comment (jplatte) [reply x.py:63]: Yes, I would prefer to use a documented functionality.

**Comment by jplatte** [comment]
@a1phyr Ping. You plan to get back to this?

**Comment by a1phyr** [comment]
Thanks for pinging me, I admit I forgot this. I get back on this !

**Comment by jplatte** [comment]
Thanks!

**Pull request merged**